frontend/src/components/LoginButton.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+
+export default function LoginButton(): any {
```

Comment:
Lets remove the any - we don't need to type components if they don't receive props (and we always use specific types instead of any), but if you want to for the sake if it you can do:

`const App: FC = () => {`